bookings/views.py

```diff
@@ -1,3 +1,39 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from uuid            import uuid4
+
+from django.db       import transaction
+from django.http     import JsonResponse
+from django.views    import View
+from bookings.models import Booking, BookingStatus, Passenger
+from users.models    import User
+
+
+
+class ReservationView(View):
+    @transaction.atomic()
+    def post(self, request):
+        try:
+            data    = json.loads(request.body)
+
+            booking = Booking.objects.create(
+                booker_name          = data['booker_name'],
+                booker_email         = data['booker_email'],
+                booker_mobile_number = data['booker_mobile_number'],
+                booking_status       = BookingStatus.objects.get(id = 1),
+                user                 = User.objects.get(id=1),
+                booking_number       = uuid4()
+            )
+
+            passengers = data['passengers']
+
+            [Passenger.objects.create(
+                first_name = passengers[f'{i}']['first_name'],
+                last_name  = passengers[f'{i}']['last_name'],
+                gender     = passengers[f'{i}']['gender'],
+                birthday = '%s-%s-%s'%(passengers[f'{i}']['year'],passengers[f'{i}']['month'],passengers[f'{i}']['day']),
+                booking_id = booking.id)for i in range(len(passengers))]
```

Comment:
1. passengers의 정보가 어떻게 들어오나요? 아래와 같이 리스트 안에 승객 한 명당 정보를 dictionary로 받으면 좋을 것 같습니다.
```python
passengers = [{},{}.{}]
```
2. 코드를 조금 정리하면 다음과 같이 수정 할 수 있을 것 같아요.
```suggestion
            passengers = data['passengers']

            passenger_list = [Passenger(
                first_name = passenger['first_name'],
                last_name  = passenger['last_name'],
                gender     = passenger['gender'],
                birthday = f'{passenger['year']}-{passenger['month']}-{passenger['day']}',
                booking_id = booking.id
            )for passenger in passengers]
            Passenger.objects.bulk_create(passenger_list)
```

Reply:
받는 데이터 수정후 코드에 반영 했습니다.